src/EnergyPlus/ExhaustAirSystemManager.cc

```diff
@@ -351,6 +334,22 @@ namespace ExhaustAirSystemManager {
 
                 thisExhSys.CentralFanIndex = centralFanIndex;
 
+
+                // std::string availSchName = ip->getAlphaFieldValue(objectFields, objectSchemaProps, "availability_schedule_name");
+                // int availSchNum = EnergyPlus::ScheduleManager::GetScheduleIndex(state, availSchName);
+                if (availSchNum > 0) {
+                    // normal conditions
+                } else if (availSchNum == 0) {
+                    // blank or anything like that, treat as always avaialabe
+                } else { // = -1 // not found a match
+                    availSchNum = 0;
+                    // a regular warning
+                    ShowWarningError(state, RoutineName + cCurrentModuleObject + "=" + thisExhSys.Name);
```

Comment:
This block isn't necessary, let the fan check it's own schedule. The exhaust system doesn't need to know what the fan's schedule is. When it calls the fan, if the schedule is off, the fan will return zero flow.